packages/dx-react-grid/src/components/table-layout.tsx

```diff
@@ -120,15 +122,16 @@ class TableLayoutBase extends React.PureComponent<TableLayoutCoreProps, TableLay
     const columns = this.getColumns();
     const minWidth = columns
       .map(column => column.width || (column.type === TABLE_FLEX_TYPE ? 0 : minColumnWidth))
-      .reduce((acc, width) => acc + width, 0);
+      .reduce((acc, width) => (acc as number) + (width as number), 0);
 
     return (
       <Layout
         {...restProps}
         tableRef={this.tableRef}
         columns={columns}
-        minWidth={minWidth}
+        minWidth={minWidth as number}
```

Comment:
Should we change Layout types to avoid using `as number` construction?

Reply:
`as number` is the result of changing `TableColumn.width` type. We allow to use string, and now must to specify type of variable, even we absolutely sure that it is a number.
In fact, `Layout.minWidth` type already have a number type
![image](https://user-images.githubusercontent.com/43685423/63440005-c5114700-c437-11e9-833a-ed9214767b3c.png)